.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature Request
+about: Suggest a new feature or enhancement.
+
+---
+<!--Please search the existing issues for relevant feature & enhancement requests and use the 👍 reaction to add upvotes to pre-existing requests.-->
+
+#### Description
+A clear and concise overview of the feature / enhancement.
+
+#### Use Cases
+Any relevant use-cases that would help us understand the use / need / value. Please share them as they can help us decide on acceptance and prioritization.
+
+#### Alternative Solutions
+A clear and concise description of any alternative solutions or features you've considered.
```

Comment:
Maybe add 'workarounds'... "any alternative solutions, workarounds or features you've considered"

Reply:
Yea, I think it makes sense - we want users to include workaround-level information. Added!